features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/logic/create.asset.js

```diff
@@ -265,7 +264,7 @@ $(function() {
 			success: function(response) {
                 var result = {};
                 try {
-                    result = JSON.parse(response);
+                    result = (typeof response == "string") ? JSON.parse(response) : response;
```

Comment:
see comment in [line 270](https://github.com/wso2/carbon-appmgt/pull/146/files#diff-63efeded844693daecc5e8fc2f47cc49R270).